package.json

```diff
@@ -209,7 +209,6 @@
     "webpack-livereload-plugin": "github:codecombat/webpack-livereload-plugin#master",
     "worker-loader": "^1.1.1"
   },
```

Comment:
Talked about this in slack - we will have to add the `dompurify` package to `dependencies` 👍

Reply:
oh ami i missing it ? then must be planty of `package-lock` error make me mistake it 🤦

Reply:
@codeluggage  oh it does contains `dompurify`, this diff doesn't show it becasue saurabh commit `dompurify` to master already